extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -0,0 +1,235 @@
+  }
+
+  /**
+   * @return {string}
+   **/
+  buildUrl_() {
+    const suffix = (this.random) ?
+        `/tokens/${this.mediaAttribute_}/interactions/random` :
+        `/interactions/${this.mediaAttribute_}/display`;
+    return encodeURIComponent(`${this.displayBaseUrl_}${suffix}`);
+  }
+
+  /**
+   * @return {!Promise<!JSONType>}
+   **/
+  queryMedia_() {
+    const url = decodeURIComponent(this.buildUrl_());
+    return xhrFor(this.win).fetchJson(url);
+  }
+
+  /** @param {string}
+   * @return {string}
+   * */
+  constructUrlFromMedia_(id) {
+    return encodeURIComponent(this.rendererBaseUrl_ + '/interaction/' + id);
+  }
+
+  /** @param {string}
+   * @return {!Element}
+   */
+  constructIframe_(src) {
+    const iframe = this.element.ownerDocument.createElement('iframe');
+    iframe.setAttribute('frameborder', '0');
+    iframe.setAttribute('allowtransparency', 'true');
+    iframe.setAttribute('scrolling', 'no');
+    iframe.src = src;
+    iframe.height = this.height_;
+    iframe.width = this.width_;
+    iframe.classList.add('apester-iframe');
+    this.applyFillContent(iframe);
+    return iframe;
+  }
+
+  /**
+   * @return {!Element}
+   */
+  constructOverflow_() {
+    const overflow = this.element.ownerDocument.createElement('div');
+    overflow.setAttribute('overflow', '');
+    overflow.className = 'apester-overflow-container';
+    const overflowButton = this.element.ownerDocument.createElement('button');
+    overflowButton.className = 'apester-overflow-button';
+    overflowButton.textContent = 'Full Size';
+    overflow.appendChild(overflowButton);
+    return overflow;
+  }
+
+  /** @override */
+  layoutCallback() {
+    this.element.classList.add('apester-container');
+    return this.queryMedia_()
+        .then(response => {
+          const media = response.payload;
+          const src = decodeURIComponent(this.constructUrlFromMedia_(
+              media.interactionId));
+          const iframe = this.constructIframe_(src);
+          const overflow = this.constructOverflow_();
+          const mutate = state => {
+            state.element.classList.add('apester-iframe-ready');
+          };
+          const state = {
+            element: iframe, mutator: mutate,
+          };
+          this.iframe_ = iframe;
+          this.element.appendChild(overflow);
+          this.element.appendChild(iframe);
+          return this.iframePromise_ = loadPromise(iframe).then(() => {
+            vsyncFor(this.win).runPromise({mutate}, state);
+            return media;
+          });
+        }, error => {
+          dev().error(TAG, 'Display', error);
+          return undefined;
+        }).then(media => {
+          this.togglePlaceholder(false);
+          let height = 0 || media.data.size.height;
+          const width = 0 || media.data.size.width;
+          height += (media.layout.directive === 'contest-poll') ? 40 : 0;
+          const amp = this.element;
+          this.iframe_.height = height;
```

Comment:
Please remove this part:
```
this.iframe_.height = height;
this.iframe_.width = width;
amp.setAttribute('height', height);
amp.setAttribute('width', width);
```

This is no longer needed - everything is done automatically by `attemptChangeHeight`.